pkg/asset/installconfig/aws/basedomain.go

```diff
@@ -61,14 +61,16 @@ func GetBaseDomain() (string, error) {
 		Message: "Base Domain",
 		Help:    "The base domain of the cluster. All DNS records will be sub-domains of this base and will also include the cluster name.\n\nIf you don't see you intended base-domain listed, create a new public Route53 hosted zone and rerun the installer.",
 		Options: publicZones,
-	}, &domain, func(ans interface{}) error {
-		choice := ans.(string)
-		i := sort.SearchStrings(publicZones, choice)
-		if i == len(publicZones) || publicZones[i] != choice {
-			return errors.Errorf("invalid base domain %q", choice)
-		}
-		return nil
-	}); err != nil {
+	},
+		&domain,
+		survey.WithValidator(func(ans interface{}) error {
+			choice := ans.(string)
+			i := sort.SearchStrings(publicZones, choice)
+			if i == len(publicZones) || publicZones[i] != choice {
+				return errors.Errorf("invalid base domain %q", choice)
+			}
+			return nil
+		})); err != nil {
```

Comment:
This indention does not look quite right. The parameters are indented at different levels.
```suggestion
		},
		&domain,
		survey.WithValidator(func(ans interface{}) error {
			choice := ans.(string)
			i := sort.SearchStrings(publicZones, choice)
			if i == len(publicZones) || publicZones[i] != choice {
				return errors.Errorf("invalid base domain %q", choice)
			}
			return nil
		}),
	); err != nil {
```